binary_clock_wallpaper.py

```diff
@@ -7,19 +9,28 @@
 result_image_path = os.path.join(base_dir, "img", "result.png")
 
 
-def draw_unit(unit, xs, y, size):
+def draw_unit(unit, xs, y, size, accent_color="#FF2536"):
     print(unit)
     dots = bin(int(unit))[2:].zfill(len(xs))
     print(dots)
     for dot, x in zip(dots, xs):
-        color = "#FF2536" if dot == "1" else "#807675"
+        color = accent_color if dot == "1" else "#807675"
         draw.rectangle((x, y, x + size, y + size), fill=color)
 
 
 time = datetime.now().strftime("%I:%M:%p")
 hour, minute, am_pm = time.split(":")
 am_pm = am_pm == "PM"
 
+accent_color = "#FF2536"
+args = sys.argv[1:]
+try:
+    if args[0] == '--color' and re.search(r'^(?:[0-9a-fA-F]{3}){1,2}$', args[1]):
+        accent_color = '#' + args[1]
+    else:
+        print('error: Invalid hex color, using default value')
+except IndexError:
+    print('error: Invalid command line arguments, using default color value')
```

Comment:
I think this should not print "Invalid command line arguments". One can run script without any argument and it should be fine. Maybe you can change it to something like "No color specified, using default color value"